wordpress-mysql/compose.yaml

```diff
@@ -22,12 +22,12 @@ services:
       - 80:80
     restart: always
     environment:
-      - WORDPRESS_DB_HOST=db
-      - WORDPRESS_DB_USER=wordpress
-      - WORDPRESS_DB_PASSWORD=wordpress
-      - WORDPRESS_DB_NAME=wordpress
+      - WORDPRESS_DB_HOST=127.0.1.1:3306
```

Comment:
The DBHost address should be on the same network as the wordpress container. Please check this [tutoria](https://www.tutorialworks.com/container-networking/)l on how to do it.